troi/cli.py

```diff
@@ -101,6 +102,24 @@ def info(patch):
         print(f"     {input['name']}, type {input['type']}: {input['desc']}")
 
 
+@cli.command()
+@click.option('-v', '--verbose', is_flag=True)
+@click.argument('files', nargs=-1)
```

Comment:
I'd prefer to just capture `@click.argument('args', nargs=-1)` and pass it wholesale to `pytest.main`

Reply:
Yes, I tried that initially, but click magic gets in the way. This goes into yet another rabbithole -- if you care about this deeply please open another PR for it.